include/x509.h

```diff
@@ -25,13 +26,13 @@ NAN_METHOD(get_issuer);
 NAN_METHOD(parse_cert);
 NAN_METHOD(verify);
 
-Local<Value> try_parse(const std::string& dataString);
-Local<Value> verify(const std::string& dataString);
+Local<Value> try_parse(const std::string &dataString);
+Local<Value> verify(const std::string &dataString);
```

Comment:
Could you please revert these style changes?

Reply:
i wish someone just edited these files using github inline edit :)